Name: Emoticons using sym+k & sym+z
Version: 1.3.5-1
Author: xanthinealkaloid
Description: Use the sym key and letter K (symbols) or Z (words) to bring up the built-in emoticons list.
Index: /usr/palm/frameworks/mojo/submissions/200.72/resources/en_us/alternatechars_table.json
===================================================================
--- /usr/palm/frameworks/mojo/submissions/200.72/resources/en_us/alternatechars_table.json
+++ /usr/palm/frameworks/mojo/submissions/200.72/resources/en_us/alternatechars_table.json
@@ -10,1 +10,1 @@
-    {"keyCode": 75, "letter":"k", "list": [":-)",":-(",";-)"]},
+    {"keyCode": 75, "letter":"k", "list": [":-)",":-(",";-)","o_O","8)",":'(",":[",":S",":!",":O","^_^","O:-)",":-*",":-D","=-@",":/",":P",":|",">:-)",">:-(","<3"]},
@@ -25,1 +25,1 @@
-    {"keyCode": 90, "letter":"z", "list": ["ž","Ž","•"]},
+    {"keyCode": 90, "letter":"z", "list": ["ž","Ž","•",":doh",":mad",":wtf",":cool",":cry",":eek",":sad",":omg",":grin",":kiss",":lol",":evil",":meh",":sick",":smile",":wink",":yuck",":eww",":heart",":angel",":redface"]},
